### ISSUE TITLE
Browsertime BrowserError: disconnected: unable to connect to renderer

### ISSUE THREAD

**Issue by chancesmith** [dreamzmaster/gulp-sitespeedio#21]
I'm getting an error this setup. What am I missing?

gulp
```javascript
/** sitespeedio: test size of project in browser **/
gulp.task('speedtest', function (done) {
  sitespeedio({
    urls: ['http://www.tcalions.com.php7-27.phx1-1.websitetestlink.com'],
    browser: 'firefox',
    depth: 0,
    browsertime: {
      connectivity: 'cable',
      iterations: 3,
      screenshot: false
    }
  })(done)
});
```

Errors:
- `Browsertime BrowserError: disconnected: unable to connect to renderer`
- `ERROR: TypeError: Cannot read property 'getItems' of undefined`

console output:
```
*[master][~/wip/tcalions/wp-content/themes/tcalions]$ gulp speedtest
[12:27:07] Using gulpfile ~/wip/tcalions/wp-content/themes/tcalions/gulpfile.js
[12:27:07] Starting 'speedtest'...
[12:27:07] Analyze your site’s web performance
[12:27:07] { urls: [ 'http://tcalions.dev' ] }
[2017-12-13 12:27:07] INFO: Versions OS: darwin 16.7.0 nodejs: v8.5.0 sitespeed.io: 5.4.0 browsertime: 1.5.2 coach: 0.34.1
[2017-12-13 12:27:07] INFO: Starting chrome for analysing http://tcalions.dev 3 time(s)
[2017-12-13 12:27:07] INFO: Testing url http://tcalions.dev run 1
[2017-12-13 12:27:08] ERROR: http://tcalions.dev generated the following error in Browsertime BrowserError: disconnected: unable to connect to renderer
  (Session info: chrome=62.0.3202.94)
  (Driver info: chromedriver=2.30.477690 (c53f4ad87510ee97b5c3425a14c0e79780cdf262),platform=Mac OS X 10.12.6 x86_64)
  (Session info: chrome=62.0.3202.94)
  (Driver info: chromedriver=2.30.477690 (c53f4ad87510ee97b5c3425a14c0e79780cdf262),platform=Mac OS X 10.12.6 x86_64)
    at startBrowser.call.catch.tap.tap.catch.catch (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/browsertime/lib/core/seleniumRunner.js:82:15)
    at tryCatcher (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/bluebird/js/release/promise.js:689:18)
    at Async._drainQueue (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:781:20)
    at tryOnImmediate (timers.js:743:5)
    at processImmediate [as _immediateCallback] (timers.js:714:5)
[2017-12-13 12:27:08] ERROR: TypeError: Cannot read property 'getItems' of undefined
    at Object.summarize (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/sitespeed.io/lib/plugins/assets/aggregator.js:101:49)
    at Object.processMessage (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/sitespeed.io/lib/plugins/assets/index.js:34:36)
    at queue.process (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/sitespeed.io/lib/support/queueHandler.js:155:57)
    at drainItem (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/concurrent-queue/index.js:92:21)
    at Immediate.drain (/Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/node_modules/concurrent-queue/index.js:73:84)
    at runCallback (timers.js:781:20)
    at tryOnImmediate (timers.js:743:5)
    at processImmediate [as _immediateCallback] (timers.js:714:5)
[2017-12-13 12:27:08] INFO: Render HTML for 1 page(s)
[2017-12-13 12:27:09] INFO: HTML stored in /Users/chancesmith/wip/tcalions/wp-content/themes/tcalions/sitespeed-result/tcalions.dev/2017-12-13-12-27-07
[2017-12-13 12:27:09] INFO: Finished analysing http://tcalions.dev
```

**Comment by dreamzmaster** [comment]
Upgraded to latest version in 'release' branch for now.
Please validate.

TODO: Some documentation still needs to be updated (please refer the latest gulpfile.js or configuration on sitespeed.io).

**Comment by dreamzmaster** [comment]
Fixed and updated to latest 2.0.0 with updated documentation.
